### PR TITLE
Add initial content

### PR DESCRIPTION
**Pull request by a-szotyori**
Add the skills challenges for iOS and Android including test invoices.

### CODE REVIEW

File: android-skills-challenge.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Good luck solving this challenge!
+
+# Requirements
+* Use Gini Bank SDK’s Screen API to take pictures of German invoices and retrieve payment information.
+* Save the payment information for each invoice. You can use a timestamp as an identifier for the invoice.
+* Show a list of all invoices (without a picture).
+* When selecting an invoice then show the saved payment information.
+
+# Resources
+* Guide: https://developer.gini.net/gini-mobile-ios/GiniBankSDK
```
Comment (zladzeyka) [inline android-skills-challenge.md]: @a-szotyori Could your please update `Resources` we have iOS here for the Android challenge?

File: ios-skills-challenge.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Good luck solving this challenge!
+
+# Requirements
+
+* Use Gini Bank SDK’s Screen API to take pictures of German invoices and retrieve payment information.
+* Save the payment information for each invoice. You can use a timestamp as an identifier for the invoice.
+* Show a list of all invoices (without a picture).
+* When selecting an invoice then show the saved payment information.
+
+# Resources
```
Comment (zladzeyka) [inline ios-skills-challenge.md:32]: @a-szotyori Could your please update `Resources` we have Android here for the iOS challenge.

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Thank you @a-szotyori 👏🏻
Could you please swap `Resouces` section for iOS and Android challenges 😉

File: android-skills-challenge.md (hunk 1 above)
Comment (a-szotyori) [reply android-skills-challenge.md]: Oh crap, thank you for spotting this! 🙏

**Review by zladzeyka — APPROVED** [review]
👏🏻👍🏻

**Pull request merged**